iam-login-service/src/main/webapp/resources/iam/apps/dashboard-app/components/clients/client/client.component.js

```diff
@@ -93,7 +135,36 @@
                         type: 'success',
                         body: 'Client saved!'
                     });
-                    $location.path('/clients');
+
+                    const isNewClientPublic = self.clientVal.token_endpoint_auth_method === 'none';
+                   
+                    if (isNewClientPublic) {
+                        $location.path('/clients');
+                        return;
+                    }
+
+                    var modalSecret = $uibModal.open({
+                        templateUrl: '/resources/iam/apps/dashboard-app/components/clients/client/newclientsecretshow/newclientsecretshow.component.html',
+                        controller: ClientSecretViewController,
+                        controllerAs: '$ctrl',
+                        resolve: {
+                            data: {
+                                client: res,
+                                title: "New client credential details",
+                                message: "Save this client credential on safe before press Confirm button",
```

Comment:
What does "New client credentials details" means?

Also: "Save this client credential on safe before press Confirm button" feels weird. I would suggest something like
"Save the client secret before confirmation, you will not be able to access it again."